src/rtems_proxy/hybrid.py

```
@@ -0,0 +1,289 @@
+    result = subprocess.run(cmd, check=False)
+    if result.returncode != 0:
+        typer.echo("ibek runtime generate-autosave failed")
+        raise typer.Exit(1)
+
+    report("ibek generate-autosave completed")
+
+
+def _run_checked(cmd: list[str], err_msg: str):
+    """Run a subprocess, converting failure to a controlled typer.Exit(1)."""
+    try:
+        result = subprocess.run(cmd, check=False)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)